### PR TITLE
added new landing page

### PR DESCRIPTION
**Pull request by jwhamilton99**

### CODE REVIEW

File: docs/css/style.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
+@font-face {
+	font-family: "JetBrains Mono";
+	src: 	url(/fonts/JetBrainsMono-Regular.woff2) format("woff2"),
```
Comment (zmknox) [inline docs/css/style.css:3]: Are we allowed to redistribute this font?

File: docs/js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+function setProjectsList(nodes) {
+	var list = document.getElementById("projects");
+	for(node of nodes) {
+		list.appendChild(node);
+	}
+}
+
+function getProjectsFromData() {
```
Comment (zmknox) [inline docs/js/scripts.js:12]: This and the below function are very similar, should we turn them into one function with some parameters?

**Review by zmknox — COMMENTED** [review]
Good work!

File: docs/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	<body onload="pageLoaded()">
+		<div id="container">
+			<div id="content">
+				<h1>We are <span id="title">The Lovely Developers</span>.</h1>
+				<p>We're a group of friends around the world who make a few things, including:</p>
+				<ul id="projects">
+					
+				</ul>
+				
+				<!-- <p id="members">We are
```
Comment (RosemaryOrchard) [inline docs/index.html:16]: Is this supposed to be here? ;)

File: docs/css/style.css (hunk 1 above)
Comment (jwhamilton99) [reply docs/css/style.css:3]: according to jetbrains: "free and open source" https://www.jetbrains.com/lp/mono/

**Review by jwhamilton99 — COMMENTED** [review]
re. JS getting data functions: i tried making that one single function that you can call with parameters, but for some reason `async` didn't pass anything to the callback so i did it a little more messy

File: docs/css/style.css (hunk 1 above)
Comment (zmknox) [reply docs/css/style.css:3]: I love this license FAQ. Right to the point https://www.jetbrains.com/lp/mono/#license

**Review by zmknox — APPROVED** [review]

**Review by RosemaryOrchard — APPROVED** [review]

**Review by benbacardi — APPROVED** [review]

**Review by dwrss — APPROVED** [review]

**Pull request merged**